TabulateSmarterTestContentPackage/Tabulator.cs

```diff
@@ -1564,7 +1564,26 @@ List<HtmlImageTagModel> ExtractImageList(XmlDocument htmlDocument)
                     Id = x.Attributes["id"]?.InnerText ?? string.Empty
                 }));
             return imgList;
-        } 
+        }
+
+        // Acceptable sub-elements: textToSpeechPronunciation, textToSpeechPronunciationAlternate, audioText, audioSortDesc, audioLongDesc
+        void CheckForNonEmptyReadAloudSubElement(ItemContext it, XmlNode xml)
+        {
+            if(new List<string> {"textToSpeechPronunciation", "textToSpeechPronunciationAlternate", "audioText", "audioShortDesc", "audioLongDesc"}
+                .Select(t => $"relatedElementInfo/readAloud/{t}") // Select sub-elements from list above
+                .Select(element => ElementExistsAndIsNonEmpty(xml, element)) // Check if the sub-element exists and has a value
+                .All(x => x == false)) // If none of the elements exist with a value 
```

Comment:
You should be able to make this a !Any() instead which is a bit cleaner plus it doesn't need to go through them all.  Just until it finds the first match.

```
if (!new List<string> {"textToSpeechPronunciation", "textToSpeechPronunciationAlternate", "audioText", "audioShortDesc", "audioLongDesc"}
                 .Select(t => $"relatedElementInfo/readAloud/{t}") // Select sub-elements from list above
                 .Any(element => ElementExistsAndIsNonEmpty(xml, element)) // Check if the sub-element exists and has a value

```